### PR TITLE
upate with fix for guestbook-frontdoor

### PR DESCRIPTION
**Pull request by beemarie**

### CODE REVIEW

**Comment by beemarie** [comment]
@linsun @duglin please take a look & review

File: workshop/exercise-5/README.md [hunk 1]
```diff
@@ -85,10 +87,29 @@ The IBM Ingress service provides IBM Cloud users with a secure, reliable, and sc
...
+2. Prepend `guestbook.` to the subdomain that you retrieved in the previous step. This new url will serve as `host` in the `guestbook-frontdoor.yaml` file, which you can find and edit in the `istio101/workshop/plans` directory.
+
+    The file should now look something like this:
+
+    ```yaml
+    apiVersion: extensions/v1beta1
+    kind: Ingress
+    metadata:
+      name: guestbook-ingress
+      namespace: istio-system
```
Comment (linsun) [inline workshop/exercise-5/README.md]: does the guestbook-ingress have to be placed in istio-system namespace?  If not, I'd prefer not to as it is a sample app.

Comment (beemarie) [reply workshop/exercise-5/README.md]: because it's forwarding to the istio-ingressgateway it needs to be in the same namespace.

Comment (linsun) [reply workshop/exercise-5/README.md]: Thanks, I compared with my old yaml, yes, it does need to be in that namespace.

**Pull request merged**